distribution/ddf-common/src/main/resources/NOTICE

```diff
@@ -110,6 +110,9 @@ This product includes software developed by
 JJ2000 partners
 Licensed under the LGPL and JJ2000 license.
 
+This software is based in part on the work of the
+Independent JPEG Group.
```

Comment:
How about mentioning the LGPL license and providing a link to somewhere on the FFmpeg site.